mappings/net/minecraft/client/util/CharPredicate.mapping

```diff
@@ -1 +1,14 @@
 CLASS net/minecraft/class_5462 net/minecraft/client/util/CharPredicate
+	METHOD method_36123 not ()Lnet/minecraft/class_5462;
```

Comment:
```suggestion
	METHOD method_36123 negate ()Lnet/minecraft/class_5462;
```
Since there is [`Predicate.negate`](https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/util/function/Predicate.html#negate()) from the jdk.